### PR TITLE
Collecting MongoDB Serverless clusters

### PR DESCRIPTION
**Pull request by AnnuCode**
## Problem

Missing collection of MongoDB Atlas Serverless clusters.

## Solution

Created a new `serverless` package to collect all the MongoDB Atlas Serverless clusters.

## Changes Made

- `serverless` package added in the mongdbatlas providers.
- `NormalizeRegionNames` function moved to `utils` package in the `regions.go` file and imported from there to be used in `serverless` and `clusters` packages.
- Test file `utils_test.go` created which tests the `NormalizeRegionNames` function.

## Checklist

- [x] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [x] Changes have been thoroughly tested
- [x] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [x] Any dependencies have been added to the project, if necessary

## Reviewers

@eneskaya

### CODE REVIEW

File: providers/mongodbatlas/sererlessinstances/serverless.go [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+				Link:       fmt.Sprintf("https://cloud.mongodb.com/v2/%s#/clusters/detail/mvp", cluster.GroupID),
+				Tags:       tags,
+			})
+		}
+	}
+
+	log.WithFields(log.Fields{
+		"provider":  "MongoDBAtlas",
+		"account":   client.Name,
+		"service":   "Cluster",
```
Comment (eneskaya) [inline providers/mongodbatlas/sererlessinstances/serverless.go:59]: Here also `"Serverless Cluster"` for service

File: utils/regions.go [hunk 2]
```diff
@@ -233,3 +235,13 @@ func GetLocationFromRegion(label string) Location {
 
 	return Location{}
 }
+
+// MongoDB Atlas returns the region names of AWS, GCP and Azure.
+// The names are written as "EU_CENTRAL_1" instead of "eu-central-1", which
+// this function fixes.
+
```
Comment (eneskaya) [inline utils/regions.go:242]: You can get rid of the new line here

**Review by eneskaya — APPROVED** [review]

**Comment by eneskaya** [comment]
Hey @AnnuCode , the PR itself looks fine. We need to do some rebasing on our master, after which you need to rebase too. Then we can merge. I'll ping you once we fixed on our side 👍

**Comment by eneskaya** [comment]
@AnnuCode please rebase your fork/PR to most recent master and we should be good to go 👍

**Comment by AnnuCode** [comment]
@eneskaya after doing `git pull --rebase` on the `master` branch, it says "Already up to date.". After that I did `git rebase master` on the `feature/618` branch and it says "Current branch feature/618 is up to date. " 


git_komiser

**Comment by eneskaya** [comment]
> @eneskaya after doing `git pull --rebase` on the `master` branch, it says "Already up to date.". After that I did `git rebase master` on the `feature/618` branch and it says "Current branch feature/618 is up to date. "
> 
> git_komiser

You may need to sync with the main repository first: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

**Comment by eneskaya** [comment]
> @eneskaya Is it possible that my past PR made by a previous fork(now deleted) is causing issues? Attaching few photos to show the conflict: First photo: Conflicts showing in the present PR Second photo: My present forked repo of Komiser showing I am 2 commits ahead(these are the same commits which are the conflicts) Third photo: The two commits which are being shown as made from the previous PR which is now closed and the last fork deleted.
> 
> **First:** git_conflct
> 
> **Second:** git_conflict1
> 
> **Third:** git_conflict3

Yeah the problem is that we needed to get rid of these two commits that you're seeing on the last screenshot. What happens when you press "Sync Fork" on the second screenshot?

**Comment by AnnuCode** [comment]
@eneskaya  It says this branch is not behind the upstream tailward:master 

sync_error

**Comment by eneskaya** [comment]
> @eneskaya It says this branch is not behind the upstream tailward:master
> 
> sync_error

Ok then do the following on `master`: `git reset --hard HEAD^` (⚠️ This will remove the top-most commit, which is the commit that we removed)
And then follow the steps from my previous [comment](https://github.com/tailwarden/komiser/pull/648#issuecomment-1481273528)

**Comment by AnnuCode** [comment]
@eneskaya  After the commands, things went like this:

git_komiser1

**Comment by eneskaya** [comment]
> @eneskaya After the commands, things went like this:
> 
> git_komiser1

Ah yes sorry for the misunderstanding. After doing the `reset`, please push instead of pull.

**Comment by eneskaya** [comment]
> @eneskaya After `git reset --hard HEAD^` from `master` I did `git push --force`. Then from the `feature/618` branch I did `git rebase master`. This removed the 2 commits ahead message from the fork repo page and shows the branch is up to date with tailwarden/komiser:master. But the conflicts regarding the two files is still there in the PR page. Any ideas if I did something wrong? 

You need to `git push -f` on your feature branch now, and we should be good to go :)

**Comment by eneskaya** [comment]
@AnnuCode Ugh, the rebasing on your feature branch didn't remove the faulty commit... Please, on your `feature/618` branch do a `git revert d911119649afe0b255c8fac1c16c032b66b97221` to remove this commit: https://github.com/AnnuCode/komiser/commit/d911119649afe0b255c8fac1c16c032b66b97221. After that do another `git push -f`.

**Comment by AnnuCode** [comment]
@eneskaya Done :) Thank you so much for all the help! I almost took a full lesson in Git from you here :smile:

**Comment by AnnuCode** [comment]
@eneskaya  Should I update the branch as well? 

update_branch

**Comment by eneskaya** [comment]
> @eneskaya Should I update the branch as well?
> 
> update_branch

Let's not mess up anything here last minute 😂 I'll merge once the checks are green ✅

**Pull request merged**